tsconfig.test.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think that two tsconfig are used like `tsconfig.json` (config for test and IDE) and `tsconfig.build.json` (for build purposes) check [this](https://github.com/kulshekhar/ts-jest/issues/889#issuecomment-442710758) comment of an issue in ts-jest